client/my-sites/stats/features/modules/stats-shares/stats-shares.tsx

```diff
@@ -138,7 +137,7 @@ const StatShares: React.FC< StatSharesProps > = ( { siteId, className } ) => {
 			{ isEmptyStateV2 && ! isLoading && ! data?.length && (
 				// show empty state
 				<StatsCard
-					className={ clsx( 'stats-card--empty-variant', className ) }
+					className={ className }
```

Comment:
Do we also need to remove the class name `stats-card--empty-variant` from [AllTimeViews](https://github.com/Automattic/wp-calypso/blob/571ba8fd01d276d0bb41d20636161841d519b608/client/my-sites/stats/sections/all-time-views-section/all-time-views-section.tsx#L96) and [StatsVideos](https://github.com/Automattic/wp-calypso/blob/571ba8fd01d276d0bb41d20636161841d519b608/client/my-sites/stats/features/modules/stats-videos/stats-videos.tsx#L88)?

Reply:
Addressed in [5355a83](https://github.com/Automattic/wp-calypso/pull/102312/commits/5355a838033c3571ade42ed7627d91c734e83f6f) and [f80ca7a](https://github.com/Automattic/wp-calypso/pull/102312/commits/f80ca7a4ffb646b08ded3e0379e032fb46ad4fb8).